geomstats/information_geometry/binomial.py

```diff
@@ -0,0 +1,178 @@
+import geomstats.backend as gs
+import geomstats.errors
+from geomstats.geometry.base import OpenSet
+from geomstats.geometry.euclidean import Euclidean
+from geomstats.geometry.riemannian_metric import RiemannianMetric
+
+from scipy.stats import binom
+
+
+class BinomialDistributions(OpenSet):
+    """Class for the manifold of binomial distributions."""
+
```

Comment:
Add something like "This is the parameter space of binomial distributions with fixed number of draws, i.e. the unit segment."